docs/rules/one-var-declaration-per-line.md

```diff
@@ -0,0 +1,85 @@
+# Require or disallow an newline around variable declarations (one-var-declaration-per-line)
+
+Some developers declare multiple var statements on the same line:
+
+```js
+var foo, bar, baz = 0;
+```
+
+Others prefer to declare one var per line.
+
+```js
+var foo,
+    bar,
+    baz = 0;
+```
+
+This rule enforces a consistent style across the entire project.
+
+## Rule Details
+
+This rule enforces a consistent coding style where newlines are required or disallowed around each var declaration inside a variable declaration statement or just when there is a variable initialization. It ignores var declarations inside for loop conditionals.
+
+### Options
+
+This rule takes one option, a string, which can be:
+
+* `"all"` enforces a newline after each variable declaration (default)
```

Comment:
"Never" would mean you must do this:

```js
var a, b, c=0;
```

I think it's a logical opposite of "always" with "initializations" being a middle ground.